features/gift_exchanges/signup_summary.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Gift Exchange Signup Summary
```

Comment:
This is definitely an improvement over the current lack of tests, but it might be good to check that the number of Offers and Requests for the fandoms are correct as well -- if those are wrong, the summary isn't as useful as it could be. If that's too big of a pain in the butt, though, this is fine as it is.

Reply:
There are a few pre-existing tests for the sign-up summary in `challenge_yuletide.feature` (which involves more sign-ups and checks the offer/request numbers). I specifically wanted to write two smaller tests just to check for the difference in behavior between the live version and the delayed version, but it's easy enough to check the numbers here, too.

Reply:
Thanks!